Testscripts/Linux/perf_mariadb.sh

```diff
@@ -0,0 +1,283 @@
+#!/bin/bash
+
+#######################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+#######################################################################
+
+#######################################################################
+#
+# perf_mariadb.sh
+# Description:
+#    Install mariadb and sysbench.
+#    Run performance test of MariaDB using sysbench.
+#    This script needs to be run on client VM.
+#
+# Supported Distros:
+#    Ubuntu/Centos/RedHat/Debian
+#    We will support SUSE and other distros in the future.
```

Comment:
Please add "TODO" here. This is the common use for future notes. Please search out TODO in the LISAv2.